src/test/unit/controllers/SjpPressListController.test.ts

```diff
@@ -0,0 +1,69 @@
+import sinon from 'sinon';
+import { Response } from 'express';
+import SjpPressListController from '../../../main/controllers/SjpPressListController';
+import fs from 'fs';
+import path from 'path';
+import { PublicationService } from '../../../main/service/publicationService';
+import {mockRequest} from '../mocks/mockRequest';
+import moment from 'moment';
+
+const rawData = fs.readFileSync(path.resolve(__dirname, '../mocks/SJPMockPage.json'), 'utf-8');
+const sjpData = JSON.parse(rawData);
+
+const rawMetaData = fs.readFileSync(path.resolve(__dirname, '../mocks/returnedArtefacts.json'), 'utf-8');
+const metaData = JSON.parse(rawMetaData)[0];
+
+const sjpPressListController = new SjpPressListController();
+
+const sjpPressListJsonStub = sinon.stub(PublicationService.prototype, 'getIndividualPublicationJson');
+const sjpPressListMetaDataStub = sinon.stub(PublicationService.prototype, 'getIndividualPublicationMetadata');
+
+const artefactId = 'abc';
+
+sjpPressListJsonStub.withArgs(artefactId).resolves(sjpData);
+sjpPressListJsonStub.withArgs('').resolves([]);
+
+sjpPressListMetaDataStub.withArgs(artefactId).resolves(metaData);
+sjpPressListMetaDataStub.withArgs('').resolves([]);
+
+const i18n = {
+  'single-justice-procedure-press': {},
+};
+
+describe('SJP Press List Controller', () => {
+  const response = { render: () => {return '';}} as unknown as Response;
+
+  it('should render the SJP press list page', () =>  {
```

Comment:
You are resolving a promise with `then` here and in the other it case you are using `async/await` can we stick to one please

Reply:
agreed. it is changed now.

Reply:
youve only changed it for the one in the comment, youve done the same thing underneath too, remember to apply the comments to all areas, dont get tunnel vision on only doing half the fix, think pragmatically and where youve made the same mistakes